comps/go/dao/query.go

```diff
@@ -1,20 +1,18 @@
 package dao
 
 import (
-	"github.com/linksoft-dev/single/comps/go/str"
+	"github.com/linksoft-dev/single/comps/go/obj"
```

Comment:
n seria bom simplificar esses imports n?
pra conseguir importar simplesmente com "comps/obj"
Assim como a gente faz com o projeto sigeflex

Reply:
need discussion on it, not sure this is a good idea